builder/scaleway/step_remove_volume.go

```diff
@@ -18,7 +18,7 @@ func (s *stepRemoveVolume) Run(ctx context.Context, state multistep.StateBag) mu
 }
 
 func (s *stepRemoveVolume) Cleanup(state multistep.StateBag) {
-	if _, ok := state.GetOk("snapshot_name"); !ok {
+	if _, ok := state.GetOk("snapshots"); !ok {
```

Comment:
This was update but not in the `Config` struct

Reply:
`snapshot_name` does not exist. It was removed alongside the snapshot step in https://github.com/scaleway/packer-plugin-scaleway/pull/89